src/components/compose.jsx

```diff
@@ -600,15 +602,40 @@ function Compose({
     const handleItems = (e) => {
       const { items } = e.clipboardData || e.dataTransfer;
       const files = [];
+      const unattachableFiles = [];
+      const unsupportedFiles = [];
       for (let i = 0; i < items.length; i++) {
         const item = items[i];
         if (item.kind === 'file') {
           const file = item.getAsFile();
-          if (file && supportedMimeTypes.includes(file.type)) {
+          if (!file) {
+            unattachableFiles.push(file);
```

Comment:
Now that I read this, it's kinda weird for `file` to even have a name (for the `alert` code below. If `!file` is true, means there's nothing? 🤔 [Possible values](https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem/kind) for `item.kind` is either `file` or `string` so this seems like dead code (it'll never run)? 🤔

Reply:
At least on Linux, file seems to have a name when I paste something or drag somethjng in...